src/msw/window.cpp

```diff
@@ -4884,6 +4884,38 @@ wxWindowMSW::MSWUpdateOnDPIChange(const wxSize& oldDPI, const wxSize& newDPI)
     // update font if necessary
     MSWUpdateFontOnDPIChange(newDPI);
 
+    // update sizers
+    if ( GetSizer() )
+    {
+        wxSizerItemList::compatibility_iterator current =
+            GetSizer()->GetChildren().GetFirst();
+        while ( current )
+        {
+            wxSizerItem* sizerItem = current->GetData();
+
+            int borderOld = sizerItem->GetBorder();
```

Comment:
Is `borderOld` really needed?

Reply:
No, its a remnant from my debugging output.